molgenis-data-mapper/src/main/resources/js/attribute-mapping.js

```diff
@@ -362,6 +479,7 @@
 
 				// Call the filterAttributeTable to only show the attributes
 				// that are explained
+				rankAttributeTable(explainedAttributes);
 				filterAttributeTable(explainedAttributes, attributes);
```

Comment:
explainedAttributes arent used anymore in the filterAttributeTable method, you can remove it from the function and call